src/cmd/MVP-4/.env

```diff
@@ -1,19 +1,19 @@
-CREDENTIALS=".\\secret.json"
+CREDENTIALS="./secret.json"
 #EXCEL_ID="1ppqYGlGTXGb-PKOcMesyvz12pLuD1XoPAJ0N_kpRJtg"
 EXCEL_ID="1BEwASubu0el9oQA6PSwVKaNU-Q6gbJ40JR6kgqguKYE"
+#EXCEL_ID="1u0cOaWQI8t7aKfZ4aQCsexKLB3rIPHqX3kDrx0QJZMU"
 EXCEL_PATH="."
 EXCEL_NAME="ade.xlsx"
-LOCAL_ADDRESS="127.0.0.1:6969"
-REMOTE_ADDRESSES="127.0.0.2:5000,127.0.0.3:5000,127.0.0.4:5000,127.0.0.5:5000,127.0.0.6:5000,127.0.0.7:5000,127.0.0.8:5000,127.0.0.9:5000"
+VEHICLE_LADDR="127.0.0.1"
 SNIFFER_LIVE="true"
-SNIFFER_DEV="\\Device\\NPF_Loopback"
+SNIFFER_DEV="lo"
```

Comment:
What does lo mean?

Reply:
is the name of the interface on my machine